main.js

```diff
@@ -99,6 +99,8 @@ ODate.prototype.update = function() {
 		dateString = ODate.format(date, 'date');
 	} else if (format === 'time-ago-limit-4-hours'){
 		dateString = ODate.timeAgo(date, { limit: 4*inSeconds.hour });
+	} else if (format !== '') {
```

Comment:
What happens if `data-o-date-format` is set to something non-sensical such as "US"?